darwin/future/meta/objects/base.py

```diff
@@ -1,20 +1,22 @@
 from __future__ import annotations
 
-from typing import Generic, List, Optional, TypeVar
+from typing import Dict, Generic, List, Optional, TypeVar
 
 from darwin.future.core.client import Client
 from darwin.future.pydantic_base import DefaultDarwin
 
 R = TypeVar("R", bound=DefaultDarwin)
+Param = Dict[str, object]
 
 
 class MetaBase(Generic[R]):
     _item: Optional[R]
     client: Client
 
-    def __init__(self, client: Client, item: Optional[R] = None) -> None:
+    def __init__(self, client: Client, item: Optional[R] = None, meta_params: Optional[Param] = None) -> None:
```

Comment:
meta_params have to be added to the MetaBase here so that chaining can happen properly with backend API calls.
`client.team.workflows.first().stages.first().push_items_to_next_stage()`

pushing items to stage requires other parameters like workflow_id, dataset_id, team_slug. This need to be passed from Meta -> Query -> Meta as it goes down the chain